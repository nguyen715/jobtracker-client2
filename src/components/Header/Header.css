@import url('https://fonts.googleapis.com/css2?family=Changa&display=swap');

.header {
  background-color: #e8e8e8;
  border-bottom: solid black 3px;
  padding-bottom: 40px;
}

h1 {
  font-family: 'Changa', sans-serif;
  font-size: 50px;
  margin: 0;
}

@media all and (min-width: 800px) {
  .header {
    display: flex;
    justify-content: space-between;
  }

  h1 {
    margin: 30px 0px 0px 50px;
  }
}

a {
  text-decoration: none;
}

a:active, a:visited, a:hover {
  color: #000;
}