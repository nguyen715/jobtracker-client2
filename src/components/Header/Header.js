import React from 'react';
import { Route, Link } from 'react-router-dom';
import './Header.css';

function Header() {
  return (
    <div className="header">
      <Link to="/">
        <h1 className="logo">Job Tracker</h1>
      </Link>
    </div>
  )
};

export default Header;