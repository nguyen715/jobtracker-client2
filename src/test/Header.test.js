import React, { useContext } from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Context from '../context/Context';

import Header from '../components/Header/Header.js';

// const context = useContext(Context);

describe('Header component', () => {
  it('renders the UI as expected', () => {
    const tree = renderer.create(
      <BrowserRouter>
        <Header />
      </BrowserRouter>
    )
    .toJSON()

    expect(tree).toMatchSnapshot()
  })
})