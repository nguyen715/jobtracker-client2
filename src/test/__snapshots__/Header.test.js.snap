// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders the UI as expected 1`] = `
<div
  className="header"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <h1
      className="logo"
    >
      Job Tracker
    </h1>
  </a>
</div>
`;
